data/json/items/comestibles/dairy.json

```diff
@@ -185,24 +164,20 @@
     "phase": "liquid",
     "fun": -12
   },
-  {
+   {
```

Comment:
Extra space fixed in subsequent commit [801e9f9](https://github.com/CleverRaven/Cataclysm-DDA/pull/50271/commits/801e9f9a4215f939bbde5d904b344c8d1051e294)